plugins/circleci/README.md

```diff
@@ -42,7 +42,7 @@ proxy:
   '/circleci/api':
     target: https://circleci.com/api/v1.1
     headers:
-      Circle-Token: ${CIRCLECI_AUTH_TOKEN}
+      circle-Token: ${CIRCLECI_AUTH_TOKEN}
```

Comment:
According to the [CircleCI REST API v1.1 docs, this header should be `Circle-Token`](https://circleci.com/docs/api/v1/index.html#authentication), with a capital C and a capital H. In addition, this is the way I know I have it in our Backstage instance and it works fine.

Just to give it a test, I tried it locally and you're right, the header does seem to be case insensitive.  I.e., local testing for me,

```
echo "With correct spelling auth:"
curl -H "Circle-Token: ${TOKEN}" https://circleci.com/api/v1.1/me > out2.txt

echo "With new lowercase spelling auth:"
curl -H "circle-Token: ${TOKEN}" https://circleci.com/api/v1.1/me > out3.txt
```

Both of these return valid.

So I think we may want to stick with the case as defined from the vendor CircleCI's best practice.  Perhaps any issue you had was unrelated per se to the capitalization, but maybe changing it in your `app-config.yaml` forced a reload to pick up a new token?

Might you be able to try it with a capital C now and see if it still works?

Reply:
Thanks, for the clarification yes it works with a capital C now, there were some other issues.